blueocean-dashboard/src/main/js/components/ColumnFilter.jsx

```diff
@@ -39,8 +43,14 @@ export class ColumnFilter extends Component {
         e.target.select();
     }
 
-    blur() {
+    blur = e => {
+        const targetElem = e.target;
```

Comment:
@michaelneale the blur handler here is not being called from the code in onChange (directly)